src/dex-ui/pages/dao/DAOProposals/CreateDAOProposal.tsx

```diff
@@ -405,14 +418,32 @@ export function CreateDAOProposal() {
         }
       }
       case DAOProposalType.TokenAssociate: {
-        const { title, description, tokenId } = data as CreateDAOTokenAssociateForm;
-        return createDAOTokenAssociateProposal({
-          title,
-          description,
-          linkToDiscussion: "", //TODO: To be removed from SC
-          daoAccountId,
-          tokenId,
-        });
+        const { title, description, tokenId, linkToDiscussion = "" } = data as CreateDAOTokenAssociateForm;
+        switch (getDAOType(currentDaoType)) {
+          case DAOType.MultiSig:
+            return createDAOTokenAssociateProposal({
+              title,
+              description,
+              linkToDiscussion: "", //TODO: To be removed from SC
```

Comment:
Since Link to discussion is required for both `MultiSig` and `GOV` we can use `linkToDiscussion` in both Proposal.

Reply:
Done.